app/__main__.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import FastAPI, HTTPException, Query
+from typing_extensions import Annotated
+from app.settings import Settings
+from httpx import AsyncClient
+
+settings = Settings()
+app = FastAPI(title="Geocoder API", version=settings.version)
+
+
+@app.get("/geocode")
+async def geocode(q: Annotated[str, Query(description="Search query")]):
+    async with AsyncClient() as client:
+        res = await client.get(f"{settings.photon_url}/api?q={q}")
+        if res.status_code == 200:
+            return res.json()
```

Comment:
There is a pydantic-package for GeoJSON FeatureCollections, but it does not include any extra keys like "country", "city" or "name". It would be possible to extend the model to add these, but I consider it too much hassle for now
https://github.com/developmentseed/geojson-pydantic